### PR TITLE
Nova chance: replace moment-mini with dayjs

### PR DESCRIPTION
**Pull request by GregWeil**
https://www.notion.so/glitch/87bda09f85184c1d81e1659aa62e4392?v=78db6cf11f8a43859bd17bc800cebde0&p=43c26630b04043d8bdab4ef65fe2dd41

I added a /shared/dayjs-convert.js that is used by both the client and server, because dayjs doesn't support moment.duration().asUnits(). The dayjs.extend() side effects don't feel great (since it hides away the relativeTime dependency from where it actually gets used), but as long as we only have a few plugins it's ok.

### CODE REVIEW

File: src/client.jsx [hunk 1]
```diff
@@ -2,8 +2,15 @@
 
 // Import Sentry early to help it initialize.
 import {configureScope} from './utils/sentry';
-
 import './polyfills.js';
+
+// Init our dayjs plugins
+import dayjs from 'dayjs';
+import relativeTimePlugin from 'dayjs/plugin/relativeTime';
+dayjs.extend(relativeTimePlugin);
```
Comment (whimsicallyson) [inline src/client.jsx]: I don't see this used anywhere?

Comment (GregWeil) [reply src/client.jsx]: It adds the dayjs().fromNow() stuff used in team-analytics-project-details

Comment (whimsicallyson) [reply src/client.jsx]: good to know!

Comment (GregWeil) [reply src/client.jsx]: I'll toss a comment there and mention that it's needed

**Comment by GregWeil** [comment]
About 15kb minified, not a huge difference but it is an improvement

**Pull request merged**